supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala

```diff
@@ -0,0 +1,468 @@
+package cromwell.backend.google.pipelines.batch
+
+import cats.data.Validated.{Invalid, Valid}
+import cats.syntax.validated._
+import com.typesafe.config.{Config, ConfigFactory}
+import common.assertion.CromwellTimeoutSpec
+import common.exception.MessageAggregation
+import cromwell.backend.google.pipelines.batch.GcpBatchTestConfig.BatchGlobalConfig
+import cromwell.backend.google.pipelines.common.PipelinesApiConfigurationAttributes._
```

Comment:
Please remove reference to common. Replace with batch configuration attributes.

Reply:
I removed references to pipelines.common in new commit